pom.xml

```diff
@@ -55,6 +55,7 @@
 
         <!-- code execution engines -->
         <module>code-execution-engines/langchain4j-code-execution-engine-graalvm-polyglot</module>
+        <module>langchain4j-mongodb-atlas</module>
```

Comment:
minor: please move it to section with embedding stores

Reply:
done